Dockerfile

```diff
@@ -6,15 +6,17 @@ ENV PIP_VERSION=22.2.2
 ENV PLONE_VERSION=6.0.0b3
 ENV EXTRA_PACKAGES="relstorage==3.4.5 psycopg2==2.9.3 python-ldap==3.4.0"
 
-RUN mkdir /wheelhouse
-
-RUN apt-get update \
-    && buildDeps="dpkg-dev gcc libbz2-dev libc6-dev libffi-dev libjpeg62-turbo-dev libldap2-dev libopenjp2-7-dev libpcre3-dev libpq-dev libsasl2-dev libssl-dev libtiff5-dev libxml2-dev libxslt1-dev wget zlib1g-dev python3-dev build-essential" \
-    && apt-get install -y --no-install-recommends $buildDeps\
-    && pip install -U "pip==${PIP_VERSION}" \
-    && rm -rf /var/lib/apt/lists/* /usr/share/doc
-
-RUN pip wheel Plone plone.volto ${EXTRA_PACKAGES} -c https://dist.plone.org/release/$PLONE_VERSION/constraints.txt  ${PIP_PARAMS} --wheel-dir=/wheelhouse
+RUN <<EOT
+    apt-get update
+    buildDeps="dpkg-dev gcc libbz2-dev libc6-dev libffi-dev libjpeg62-turbo-dev libldap2-dev libopenjp2-7-dev libpcre3-dev libpq-dev libsasl2-dev libssl-dev libtiff5-dev libxml2-dev libxslt1-dev wget zlib1g-dev python3-dev build-essential"
+    apt-get install -y --no-install-recommends $buildDeps
+    rm -rf /var/lib/apt/lists/* /usr/share/doc
+    python -m venv /venv
+    /venv/bin/pip install -U "pip==${PIP_VERSION}"
+    /venv/bin/pip install Plone plone.volto ${EXTRA_PACKAGES} -c https://dist.plone.org/release/$PLONE_VERSION/constraints.txt  ${PIP_PARAMS}
+    rm .cache -rf
+    find /venv \( -type f -a -name '*.pyc' -o -name '*.pyo' \) -exec rm -rf '{}' +
+EOT
```

Comment:
@jensens We should double check that an error in any command still aborts the build when using the heredoc syntax

Reply:
it does :D - I just run into it